Assets/Scripts/Screeps3D/RoomObjects/Views/ObserverView.cs

```diff
@@ -0,0 +1,39 @@
+﻿using Common;
+using UnityEngine;
+
+namespace Screeps3D.RoomObjects.Views
+{
+    public class ObserverView : MonoBehaviour, IObjectViewComponent
+    {
+        public const string Path = "Prefabs/RoomObjects/observer";
+        [SerializeField] private Renderer _eye = default;
+        [SerializeField] private Renderer _spire = default;
+        private Observer _observer;
+       
+        public void Init()
+        {
+        }
+
+        public void Load(RoomObject roomObject)
+        {
+            _observer = roomObject as Observer;
+            Color c = _observer.Owner.UserId.Equals(Screeps_API.ScreepsAPI.Me.UserId) ?  new Color(0.33f, 1.000f, 0.33f, 0.0f) : new Color(1.000f, 0.33f, 0.33f, 0.0f);
```

Comment:
if  this is an owned color and an enemy color, should we not put that into a constant so it is the same everywhere?